src/qml/RelationCombobox.qml

```diff
@@ -114,7 +114,7 @@ Item {
             featureListModel.searchTerm = searchField.displayText
           }
 
-          Keys.onPressed: {
+          Keys.onPressed: (event)=> {
```

Comment:
@domi4484 , thanks for hunting these, the more we can fix now, the better we'll be when Qt decided to kill injected parameters altogether (which I hope will be sooner than later! :) )